packages/common/src/ticker.ts

```diff
@@ -59,13 +59,23 @@ export class Ticker {
     let ticker = document.createElement("div");
     ticker.classList.add("en__component");
     ticker.classList.add("en__component--ticker");
-    let str = '<div class="ticker">';
+    let str = `<div class="ticker">`;
     for (let i = 0; i < items.length; i++) {
       str += '<div class="ticker__item">' + items[i] + "</div>";
     }
     str = '<div id="engrid-ticker">' + str + "</div></div>";
     ticker.innerHTML = str;
+
     this.tickerElement?.parentElement?.insertBefore(ticker, this.tickerElement);
     this.tickerElement?.remove();
+    
+    const tickerWidth = (document.querySelector(".ticker") as HTMLElement).offsetWidth.toString();
+    ticker.style.setProperty("--ticker-size", tickerWidth);
+    
+    if(ENGrid.debug) {
```

Comment:
@cryptidgit Last change before merging, I promise :)
If you are using `this.logger.log`, you don't need to check for the `ENGrid.debug`. The logger already does the debug checking.

Reply:
Oh, that's good to know!